### PR TITLE
CA Notification web component

### PR DESCRIPTION
**Pull request by norisalsaadie**
This commit adds ca-notification and ca-notification-container.

New components:
components/ca-notification.css
components/ca-notification.js
components/ca-notification-container.css
components/ca-notification-container.js

New example
examples/ca-notification-container.html

### CODE REVIEW

**Comment by kevmarchant** [comment]
Really nice and clean code.  +1 from me.

File: components/ca-notification.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  color: #a94442;
+  background-color: #f2dede;
+  border-color: #ebccd1;
+}
+
+
+ca-notification .ca-notification-close {
+  float: right;
+  opacity: 0.2;
+  text-decoration: none !important;
```
Comment (joeburton) [inline components/ca-notification.css]: best to remove the !important.

File: components/ca-notification-container.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        * closeStack - get a list of defered close functions.
+        * @param {string} value - timeout override
+        * @return {array} list of defered close functions.
+        */
+        set timeout(value) {
+            this.getAttribute('timeout', value);
+        }
+
+        /**
+        * addNotification - function to manage the hiding of the notification
```
Comment (joeburton) [inline components/ca-notification-container.js]: update comment..

File: components/ca-notification-container.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        * closeStack - get a list of defered close functions.
+        * @return {array} list of defered close functions.
+        */
+        get closeStack() {
+            return this._closeStack;
+        }
+
+        /**
+        * timeout - get timeout that overrides notification timeouts
+        * @return {string} list of defered close functions.
```
Comment (joeburton) [inline components/ca-notification-container.js]: update comment...

File: components/ca-notification.css (hunk 1 above)
Comment (kevmarchant) [inline components/ca-notification.css]: oh good catch!

**Comment by norisalsaadie** [comment]
@joeburton changes committed.

**Pull request merged**